classification/models/condition_text_matching.py

```diff
@@ -1063,6 +1063,28 @@ def condition_matching_suggestions(ct: ConditionText, ignore_existing=False) ->
     return suggestions
 
 
+def raise_condition_resolution_flag(classification: Classification, condition_resolution):
+    classification_data = Classification.objects.filter(id=classification.id).values('condition_resolution').first()
+    old_condition_resolution = classification_data['condition_resolution']
+    condition_text_old = "No Condition"
+    if old_condition_resolution:
+        condition_text_old = ', '.join(term['term_id'] for term in old_condition_resolution.get('resolved_terms', []))
```

Comment:
Rather than looking at the JSON directly, cleaner to turn the JSON into a ConditionResolved via
ConditionResolved.from_dict(cr_dict)
that way you can use the same summary method on the old and new